### PR TITLE
Gdomingu/add pagination pets admin

### PR DESCRIPTION
**Pull request by gdomingu**
# 🔗 Issue
Fixes https://github.com/rubyforgood/pet-rescue/issues/644

# ✍️ Description
Uses Pagy gem to add pagination to the admin pets page.
Also increases the amount of pets seeded with the DB. 
co-authored with @tpope and @greggroth

# 📷 Screenshots/Demos
Screenshot of pagination controls

### CODE REVIEW

**Review by mononoken — APPROVED** [review]
Looks great! Glad to see the Pagy implementation is clean and simple. Thanks for the help @gdomingu and co!

**Comment by mononoken** [comment]
Spoke to soon. There are a few lint errors. Please run `standardrb --fix` @gdomingu . Thank you!

**Review by mononoken — CHANGES_REQUESTED** [review]
Spoke to soon. There are a few lint errors. Please run `standardrb --fix` @gdomingu . Thank you!

**Comment by kasugaijin** [comment]
@all-contributors
please add @gdomingu for code.

**Comment by kasugaijin** [comment]
@all-contributors
please add @tpope for code. 
please add @greggroth for code.

File: db/seeds/01_alta.rb [hunk 1]
```diff
@@ -228,7 +228,7 @@
   end
 
   path = Rails.root.join("app", "assets", "images", "hero.jpg")
-  10.times do
+  50.times do
```
Comment (kasugaijin) [inline db/seeds/01_alta.rb:231]: Could we do the same for the baja seed file, also? Doesn't have to be 50, but more than 10 so pagination shows. Thanks!

**Review by kasugaijin — APPROVED** [review]
Looks good to me! Played with it and works well. I did not realize how simple Pagy gem makes this. Please, just one small tweak to create more Pets in the baja seed file. Thanks.

File: db/seeds/01_alta.rb (hunk 1 above)
Comment (gdomingu) [reply db/seeds/01_alta.rb:231]: done!

Comment (kasugaijin) [reply db/seeds/01_alta.rb:231]: Thank you!

**Pull request merged**